### PR TITLE
Fix cache and check scan

### PR DESCRIPTION
**Pull request by sorhawell**
aims to solve #28

### CODE REVIEW

**Comment by sorhawell** [comment]
still missing to set new action env_var RPOLARS_RUST_SOURCE in workflow

**Comment by sorhawell** [comment]
wrong zstd in path error  https://github.com/actions/cache/issues/1073

File: src/Makevars.win [hunk 1]
```diff
@@ -14,6 +14,11 @@ rpolars_ext_binary = "$(RPOLARS_RUST_SOURCE)/target/$(TARGET)/release/librpolars
 all: C_clean
 
 $(SHLIB): $(STATLIB)
+# CRAN policy forbids using $HOME so we build in the current working dir.
+# This makes things really slow because we have to reinstall all dependencies
+# for each build. Comment this out for non-CRAN releases.
+CARGOTMP=$(PWD)/.cargo
```
Comment (eitsupi) [inline src/Makevars.win]: `$(PWD)` may not work here.
`$(CURDIR)` should be use in Makefiles.
https://stackoverflow.com/questions/52437728/bash-what-is-the-difference-between-pwd-and-curdir

Comment (sorhawell) [reply src/Makevars.win]: Thank you for seeing this,  I will revisit this next week

**Pull request merged**